src/Microsoft.OData.Edm/Vocabularies/CoreVocabularies.xml

```diff
@@ -443,6 +443,14 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
     <Annotation Term="Core.LongDescription" String="The annotation value will usually be an expression, e.g. using properties of the binding parameter type for instance-dependent availability, or using properties of a singleton for global availability. The static value `null` means that availability cannot be determined upfront and is instead expressed as an operation advertisement." />
   </Term>
 
+  <Term Name="RequiresExplicitBinding" Type="Core.Tag" DefaultValue="true" AppliesTo="Action Function">
+	<Annotation Term="Core.Description" String="This bound action or function is only available on model elements annotated with the ExplicitOperationBindings term." />
+  </Term>
+
+  <Term Name="ExplicitOperationBindings" Type="Collection(Core.QualifiedBoundOperationName)">
+    <Annotation Term="Core.Description" String="The qualified names of explicitly bound operations that are supported on the target model element. These operations are in addition to any operations not annotated with RequiresExplicitBinding that are bound to the type of the target model element." />
+  </Term>

```

Comment:
Note that, in the final version of the OASIS PR, the Description was updated and a LongDescription was added.